dev_sandbox/data_exploration_1D.py

```diff
@@ -0,0 +1,13 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+rel_data_path = "data/X001_droplet_amplitudes.csv" # make sure you are in the very top directory of the project! 
+dframe = pd.read_csv(rel_data_path, header=0)
+print(dframe.head()) # take a look at the data
```

Comment:
suggestion: Python style guide suggests 2 spaces before a comment when inline with code.

Reply:
Should be edited now.